packages/tooling/fast-tooling-wasm/app/index.ts

```diff
@@ -0,0 +1,20 @@
+import NumberSchema from "./schemas/number.schema.json";
+
+/**
+ * Ensure the global Module variable is known
+ * This is created in the permutator.js file
+ */
+declare var Module: any;
+
+console.log("JSONSchema", NumberSchema);
+
+var permutate = Module.cwrap("permutate", "number", ["number"]);
```

Comment:
Not having looked at the permutator.js file, I wonder if we could hand-author that in the future so that there's a nicer import syntax without having to force the consumer to fiddle with `Module`. Just a thought for the future.

Reply:
For sure, I'll put that in my next PR